openapi.yaml

```diff
@@ -4408,8 +4408,6 @@ components:
           maxItems: 64
           minItems: 1
           type: array
-      required:
```

Comment:
Let's include extra docs about the meaning of empty and missing/null `repositories`, similar to releases.

Reply:
added to `repositories` field with allOf/$ref-description trick